algolia/internal/gen/options.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
While I agree that this is how new tags should be written, do we need this?
I believe the previous syntax (on the line below) is still valid go.

Reply:
No worries, just wanted to make sure I wasn't missing something overly subtle.